homeassistant/components/elkm1/__init__.py

```diff
@@ -83,17 +83,26 @@ def _elk_value(val):
             vol.Required(CONF_HOST): cv.string,
             vol.Optional(CONF_USERNAME, default=''): cv.string,
             vol.Optional(CONF_PASSWORD, default=''): cv.string,
-            vol.Optional(CONF_TEMPERATURE_UNIT, default=TEMP_FAHRENHEIT):
+            vol.Optional(CONF_TEMPERATURE_UNIT, default='F'):
                 cv.temperature_unit,
-            vol.Optional(CONF_AREA): CONFIG_SCHEMA_SUBDOMAIN,
-            vol.Optional(CONF_COUNTER): CONFIG_SCHEMA_SUBDOMAIN,
-            vol.Optional(CONF_KEYPAD): CONFIG_SCHEMA_SUBDOMAIN,
-            vol.Optional(CONF_OUTPUT): CONFIG_SCHEMA_SUBDOMAIN,
-            vol.Optional(CONF_PLC): CONFIG_SCHEMA_SUBDOMAIN,
-            vol.Optional(CONF_SETTING): CONFIG_SCHEMA_SUBDOMAIN,
-            vol.Optional(CONF_TASK): CONFIG_SCHEMA_SUBDOMAIN,
-            vol.Optional(CONF_THERMOSTAT): CONFIG_SCHEMA_SUBDOMAIN,
-            vol.Optional(CONF_ZONE): CONFIG_SCHEMA_SUBDOMAIN,
+            vol.Optional(CONF_AREA, default={CONF_ENABLED: True}):
```

Comment:
An empty dict would also work as default value, since the default value is passed to the `CONFIG_SCHEMA_SUBDOMAIN` where the defaults of the dict are populated.

Reply:
Thank you! I made the change you suggested.